assets/style/style.css

```diff
@@ -92,6 +94,72 @@ body {
   border-bottom-left-radius: 5rem;
 }
 
+#humberger {
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  width: 100%;
+  right: -100%;
+  background: hsla(234, 100%, 69%, 90%);
+  backdrop-filter: blur(100em);
+  -webkit-backdrop-filter: blur(100em);
+  mix-blend-mode: multiply;
+  padding: 2rem 2rem 0.6rem 2rem;
+  gap: 3rem;
+  transition: all 0.5s ease;
+  opacity: 0;
```

Comment:
- [ OPTIONAL ] In my opinion it would be better to make sure the background of your headline section looks like the Figma (#fff) and your hamburger button is blue.

![Screenshot from 2023-03-07 15-03-05](https://user-images.githubusercontent.com/97834160/223430601-8c13f0f2-a592-496c-a12e-e3bde900881c.png)

Reply:
Thank you for your feedback @GedeonTS , can you try to check it well ? because I made it as you mention maybe if is still not working please lemme know in which browser you are testing I might check for compatibility.


![image](https://user-images.githubusercontent.com/101065560/223433715-6b0a3a86-1919-450f-ac73-76315484b905.png)

Regards